utils/posts.js

```diff
@@ -3,6 +3,7 @@ import path from 'path';
 import matter from 'gray-matter';
 import remark from 'remark';
 import html from 'remark-html';
+import _ from 'lodash';
```

Comment:
I would prefer to use `_` (lodash) for all of our iteration and functional-style/higher-order functions, but if we aren't going to use it, let's just take out the import